pulp_smash/pulp3/pytest_plugin/newhooks.py

```diff
@@ -0,0 +1,4 @@
+def pytest_check_for_leftover_pulp_objects(config):
```

Comment:
Is this filename a common one, or taken from an example? Should we name it `smashhooks` or `pulphooks`?